### PR TITLE
consolidating flex samples into single project

### PR DESCRIPTION
**Pull request by jabubake**
WIP

### CODE REVIEW

File: flexible/java/samples/src/main/appengine/app.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+env: flex
+
+handlers:
+- url: /.*
+  script: this field is required, but ignored
+
+use_memcache_proxy: true
+
+# [START env_variables]
+env_variables:
```
Comment (lesv) [inline flexible/java/samples/src/main/appengine/app.yaml]: This is probably how it's going to stay, but it's nice if we can do variable substitution `${blah}`, but I don't see a way to support gradle with that.

Comment (lesv) [inline flexible/java/samples/src/main/appengine/app.yaml]: What I like to do is provide credentials at build time `mvn package -DSENDGRID_API_KEY=bha`

File: flexible/java/samples/src/main/java/com/example/flexible/SamplesIndex.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import javax.servlet.http.HttpServletRequest;
+
+public class SamplesIndex {
+
+  private static Set<String> ignorePatterns = new HashSet<>();
+
+  static {
+    ignorePatterns.addAll(Arrays.asList("jsp", "default", "cron"));
+  }
+
```
Comment (lesv) [inline flexible/java/samples/src/main/java/com/example/flexible/SamplesIndex.java]: This is nice, but you might wish to do it as a jsp.

Comment (lesv) [inline flexible/java/samples/src/main/java/com/example/flexible/SamplesIndex.java]: Or in a Servlet.

File: flexible/java/samples/src/main/webapp/disk.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<html>
+<title>Disk Sample</title>
+<body>
+<form method="GET" action="/disk">
+  <p>Track user ip on disk
+    <input type="submit" value="Submit" />
+</form>
+</body>
+</html>
```
Comment (lesv) [inline flexible/java/samples/src/main/webapp/disk.html]: new line

File: flexible/java/samples/src/main/webapp/index.jsp [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    %>
+    <li><%= sample %>
+    </li>
+    <%
+        }
+    %>
+    <li><a href="/static.html">A static HTML page</a></li>
+</ul>
+</body>
+</html>
```
Comment (lesv) [inline flexible/java/samples/src/main/webapp/index.jsp]: newLine

File: flexible/java/samples/src/main/webapp/memcache.html [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+  <meta charset="UTF-8">
+  <title>Memcache test</title>
+</head>
+<body>
+<form method="GET" action="/memcache">
+  <input type="submit" value="Memcache sample" />
+</form>
+</body>
+</html>
```
Comment (lesv) [inline flexible/java/samples/src/main/webapp/memcache.html]: newLine

File: flexible/java/samples/src/main/webapp/sendgrid/email/send.html [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<html>
+<title>Send an email using SendGrid</title>
+<body>
+<form method="GET" action="/sendgrid/email/send">
+  <p>Enter an email address : </p>
+  <input type="email" name="to">
+  <input type="submit" value="Submit" />
+</form>
+</body>
+</html>
```
Comment (lesv) [inline flexible/java/samples/src/main/webapp/sendgrid/email/send.html]: newLine

File: flexible/java/samples/src/main/webapp/twillio/sms/receive.html [hunk 7]
```diff
@@ -0,0 +1,8 @@
+<html>
+<title>Twilio Code Samples</title>
+<body>
+<form method="POST" action="/twillio/sms/receive">
+  <input type="submit" value="Receive an SMS" />
+</form>
+</body>
+</html>
```
Comment (lesv) [inline flexible/java/samples/src/main/webapp/twillio/sms/receive.html]: newLine

File: flexible/java/samples/src/main/webapp/twillio/sms/send.html [hunk 8]
```diff
@@ -0,0 +1,9 @@
+<html>
+<title>Twilio Code Samples</title>
+<body>
+<form method="GET" action="/twillio/sms/send">
+  <p>Enter a name : <input type="text" name="to">
+  <p><input type="submit" value="Send an SMS" />
+</form>
+</body>
+</html>
```
Comment (lesv) [inline flexible/java/samples/src/main/webapp/twillio/sms/send.html]: newLine

**Comment by jabubake** [comment]
Closing this pull request.

**Pull request closed without merging**